src/app/admin/analytics/analytics.component.html

```diff
@@ -36,7 +36,7 @@
           <span>Nothing found</span>
         </div>
         <md-list *ngIf="users && users.admins">
-          <a *ngFor="let user of users.admins" class="online-row" [routerLink]="['/profile', user._id]">
+          <a *ngFor="let user of users.admins" class="online-row" [routerLink]="['/profile', user.username]">
```

Comment:
Any profile link of an user that has not set his username yet, will route to `/profile/undefined`.  Is that acceptable?